workloads/ingress-perf/config/aws-standard.yml

```diff
@@ -0,0 +1,55 @@
+# vi: expandtab shiftwidth=2 softtabstop=2
+
+# First scenario is configured as warmup and it will also tune the default ingress-controller to assign the router pods to the infra nodes
+- termination: http
+  connections: 200
+  samples: 4
+  duration: 5m
+  path: /1024.html
+  concurrency: 18
+  tool: wrk
+  serverReplicas: 45
+  tuningPatch: '{"spec":{"nodePlacement": {"nodeSelector": {"matchLabels": {"node-role.kubernetes.io/infra": ""}}}, "replicas": 2}}'
+  delay: 10s
+  warmup: true
+
+- termination: http
+  connections: 200
+  samples: 2
```

Comment:
From your testing, did you find 2 samples is enough?

Reply:
yeah, but let me bump it to 3 samples just in case